src/shared/components/mutationTable/MutationTableUtils.ts

```diff
@@ -1,91 +1,31 @@
-                    namespaceName,
-                    namespaceColumnName
-                );
-                const fRender =
-                    type === 'number'
-                        ? (d: Mutation[]) =>
-                              NumericNamespaceColumnFormatter.renderFunction(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              )
-                        : (d: Mutation[]) =>
-                              CategoricalNamespaceColumnFormatter.renderFunction(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              );
-                const fSort =
-                    type === 'number'
-                        ? (d: Mutation[]) =>
-                              NumericNamespaceColumnFormatter.sortValue(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              )
-                        : (d: Mutation[]) =>
-                              CategoricalNamespaceColumnFormatter.sortValue(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              );
+    > = createNamespaceColumns<Mutation>(namespaceColumnConfig);
 
-                const fDownload =
-                    type === 'number'
-                        ? (d: Mutation[]) =>
-                              NumericNamespaceColumnFormatter.download(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              )
-                        : (d: Mutation[]) =>
-                              CategoricalNamespaceColumnFormatter.download(
-                                  d,
-                                  namespaceName,
-                                  namespaceColumnName
-                              );
-                columns[columnName] = {
-                    id: columnName,
-                    name: columnName,
-                    render: fRender,
-                    sortBy: fSort,
-                    download: fDownload,
-                    filter: (
-                        d: Mutation[],
-                        filterString: string,
-                        filterStringUpper: string
-                    ) => defaultFilter(d, columnName, filterStringUpper),
-                    visible: !!getServerConfig()
-                        .skin_mutation_table_namespace_column_show_by_default,
-                    order: 400,
-                    align: type === 'number' ? 'right' : 'left',
-                };
-            })
-        );
+    for (const key in columns) {
```

Comment:
just use lodash ```_.forEach( (column, key)=>{ ... } )```

Reply:
Fixed in [515fec4](https://github.com/cBioPortal/cbioportal-frontend/pull/4418/commits/515fec4546c8becb2c90691991dc6ee44973ffe3)